build-tools/knnplugin-coverage.gradle

```diff
@@ -43,6 +43,7 @@ task dummyIntegTest(type: Test) {
 
 integTest {
     systemProperty 'jacoco.dir', "${jacocoDir}"
+    systemProperty 'jmxServiceUrl', "service:jmx:rmi:///jndi/rmi://127.0.0.1:7777/jmxrmi"
```

Comment:
should we use a "." delimiter like "jmx.serviceUrl" to make it consistent with existing properties?

Reply:
Ack